collectors/ebpf_process.plugin/ebpf_process.c

```diff
@@ -511,13 +511,13 @@ void *process_publisher(void *ptr)
 }
 
 static void move_from_kernel2user_global() {
-    uint32_t idx;
-    uint32_t res[NETDATA_GLOBAL_VECTOR];
+    uint64_t idx;
+    uint64_t res[NETDATA_GLOBAL_VECTOR];
```

Comment:
makes sense to typedef it in common hdr?

Reply:
I am not sure whether I understood your question, I arrived in the conclusion that you are talking about `NETDATA_GLOBAL_VECTOR` definition. This is also a static value inside the `eBPF program` this is the motive I used the definition here and inside the eBPF code.  Case we bring more kprobes one day, it will need to change both sides.

Reply:
After we talk on Slack I understood better what you were proposing and I brought two commits to convert the code in more readable version.